CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.10.0] - 2024-03-21
```

Comment:
Got this number from `version.txt`. Do I have to update `version.txt` or is this good?